python/lsst/ts/m2gui/controltab/tab_cell_status.py

```diff
@@ -33,7 +43,282 @@ class TabCellStatus(TabDefault):
         Table's title.
     model : `Model`
         Model class.
+
+    Attributes
+    ----------
+    model : `Model`
+        Model class.
     """
 
     def __init__(self, title, model):
         super().__init__(title, model)
+
+        self._view_mirror = ViewMirror()
+        self._button_show_alias = set_button(
+            "Show Alias", self._callback_show_alias, is_checkable=True
+        )
+
+        self._forces = ActuatorForce()
+        self._figures = self._create_figures()
+
+        self._actuator_data_selection = self._create_actuator_data_selection()
+
+        # Internal layout
+        self.widget().setLayout(self._create_layout())
+
+        # Resize the dock to have a similar size of layout
```

Comment:
Resize the dock to match layout size

Reply:
Fixed in the commit: [af8c4ea](https://github.com/lsst-ts/ts_m2gui/pull/13/commits/af8c4eada9478d63f692c5cd50c53c057e3e2b34). Thanks!